modules/programs/helix.nix

```
@@ -161,7 +168,22 @@ in {
   };
 
   config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
+    home.packages = if cfg.extraPackages != [ ] then
+      [
+        (pkgs.symlinkJoin {
+          name = "helix-wrapped-${lib.getVersion pkgs.helix}";
+          paths = [ pkgs.helix ];
```

Also, so that the package configuration gets passed on:

```suggestion
          name = "${lib.getName cfg.package}-wrapped-${lib.getVersion cfg.package}";
          paths = [ cfg.package ];
```